### ISSUE TITLE
questions about render_image.py

### ISSUE THREAD

**Issue by cezarbbb** [cmusatyalab/mega-nerf#51]
@hturki thanks for your great work! I'm following the tutorial to see its outcome. However, after the evaluation, I can only see the matrixs.txt, which I want is to see some rendered images or videos. The question is I don't really understand how to creat 3 files of the input param in render_image.py, could you pls offer some examples?

**Comment by cezarbbb** [comment]
At present I have succeeded in rendering an image using poses, intrinsics and embeddings provided by the train dataset(those pt files). However I modified line 95 to make it work. The original line is "ImageMetadata(Path(''), c2ws[i], W, H, torch.FloatTensor(intrinsics[i][2:]), embeddings[i], None, False)". I deleted the number 2 in the intrinsics matrix, so the modified line is "ImageMetadata(Path(''), c2ws[i], W, H, torch.FloatTensor(intrinsics[i][:]), embeddings[i], None, False)" and that works. I'm not sure if that is really a mistake?

**Comment by cezarbbb** [comment]
I write a new script in my fork in order to generate video by using _render_images function
in render_images.py provided and the train data(those pt files including poses and intrinsics). The mega nerf viewer need too much vram to run it :(

**Comment by heye0507** [comment]
> @hturki thanks for your great work! I'm following the tutorial to see its outcome. However, after the evaluation, I can only see the matrixs.txt, which I want is to see some rendered images or videos. The question is I don't really understand how to creat 3 files of the input param in render_image.py, could you pls offer some examples?

You need to generate poses by yourself. Here is a quick example:

a) For poses, you can randomly start at one location in your scene. A quick easy way is you choose one pose from the training meta.pt file. The first 3 by 3 is rotation matrix, you can rotate it with the rotation matrix R. For example, [1,0,0; 0, cos, -sin;0, sin, cos] will turn your first axis by theta degree. The latter 3 by 1 vector is the transition matrix, you can shift it via moving z (height), x(horizontal), y(vertical).

This is how you create a new pose in the scene, but mega-nerf is bounded by the unit sphere from nerf ++, so make sure the new pose is within the range. 

2) intrinsic you can just copy the training meta.pt if you use pretrained model

3) embedding needs to have the same length as your pose / intrinsic file. for example, it can be 0 to 10 if you have 11 poses.
